ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -63,8 +63,19 @@ gcp_connection::list_objects(std::vector<std::string> &objects, bool list_single
 
 // Puts an object into a google cloud bucket.
 int
-gcp_connection::put_object(const std::string &object_key, const std::string &file_name) const
+gcp_connection::put_object(const std::string &object_key, const std::string &file_path)
 {
+    // Note that the client library automatically computes a hash on the
+    // client-side to verify data integrity during transmission.
+    google::cloud::v2_5_0::StatusOr<gcs::ObjectMetadata> metadata =
+      _gcp_client.UploadFile(file_path, _bucket_name, _object_prefix + object_key);
+
+    // Check if file has been successfully uploaded.
+    if (!metadata) {
+        std::cout << "Upload failed: " << metadata.status() << std::endl;
```

Comment:
nit: In `list_objects` we print the error to `cerr` but here it's `cout`. I know we're replacing all these in WT-10368 but it's worth keeping them consistent until that happens.

Reply:
sounds good, it has been addressed here: [decd3ee](https://github.com/wiredtiger/wiredtiger/pull/8730/commits/decd3ee36a0f8580d15d981583b087d71d65531c) :))